NthKey.xcodeproj/project.pbxproj

```diff
@@ -708,16 +708,16 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 8;
 				DEVELOPMENT_ASSET_PATHS = "\"NthKey/Preview Content\"";
-				DEVELOPMENT_TEAM = 227MFQPYQ4;
+				DEVELOPMENT_TEAM = ERCG8ZPCT3;
 				ENABLE_PREVIEWS = YES;
 				INFOPLIST_FILE = "$(SRCROOT)/NthKey/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 14.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 0.0.8;
-				PRODUCT_BUNDLE_IDENTIFIER = com.nthkey.app;
+				PRODUCT_BUNDLE_IDENTIFIER = com.nthkey.app.test;
```

Comment:
Where does bundle identifier change take effect? Just on the simulator and device when doing development? Or also on TestFlight?

Reply:
This will be reverted (Y)